frontend/app/src/components/settings/explorers/Explorers.vue

```diff
@@ -76,6 +85,26 @@ const saveTransaction = async (newTransaction?: string) => {
   });
 };
 
+const saveBlock = async (newBlock?: string) => {
+  const setting = get(explorers)[get(selection)];
+
+  const updated = {
+    ...setting,
+    block: newBlock
+  };
+
+  if (!newBlock) {
+    delete updated.block;
+  }
+
+  await store.updateSetting({
+    explorers: {
+      ...get(explorers),
+      [get(selection)]: updated
+    }
+  });
+};
```

Comment:
do you think we can somehow unify these 3 methods by making a new function that take the new value and the key? they seem almost the same with only the update key being different